kasa/transports/sslaestransport.py

```diff
@@ -292,6 +303,34 @@ async def send_secure_passthrough(self, request: str) -> dict[str, Any]:
                 ) from ex
         return ret_val  # type: ignore[return-value]
 
+    async def send_unencrypted(self, request: str) -> dict[str, Any]:
+        """Send encrypted message as passthrough."""
+        url = cast(URL, self._token_url)
+
+        _LOGGER.debug(
+            "Sending unencrypted from %s",
```

Comment:
```suggestion
            "Sending unencrypted to %s",
```
Maybe? Should this also log the payload, or maybe it's enough if http client does it?

Reply:
The http client logs so should be fine